src/utils/api/helper.ts

```diff
@@ -1360,3 +1408,22 @@ export const evaluateFeatureFlag = async (
     )
   )();
 };
+
+export const checkLogout = (onLogoutCallBack: () => void) => {
```

Comment:
you can try in order to avoid disable eslint rule
```suggestion
export const checkLogout = async (onLogoutCallBack: () => void) => {
  await pipe(
    SessionItems.authToken,
    O.fromNullable,
    O.fold(
      () => undefined,
      () => logoutUser({ onError: onLogoutCallBack, onResponse: onLogoutCallBack })
    )
  );
};
```